examples/fetch-game-list-example-external/src/App.js

```diff
@@ -38,12 +36,14 @@ class App extends Component {
     };
   }
   componentDidMount() {
-    getGames().then(result => {
-      this.setState(state => ({
-        games: result,
-        isLoading: false
-      }));
-    });
+    fetch(`${BASE_URL}/games`)
```

Comment:
if you could move this back to the getGames() that got imported, it'd be great - otherwise the changes look good

Reply:
Awesome will do!